### DISCUSSION TITLE
Error in addDoubletScores()

### DISCUSSION THREAD

**Discussion by Tutu-Shi** [GreenleafLab/ArchR#745] [Questions / Documentation]
Hi ArchR team,

I'm following [the tutorial](https://www.archrproject.com/articles/Articles/tutorial.html) to start a new journey with the tutorial data. But something went wrong in this step:
`doubScores <- addDoubletScores(
  input = ArrowFiles,
  k = 10, #Refers to how many cells near a "pseudo-doublet" to count.
  knnMethod = "UMAP", #Refers to the embedding to use for nearest neighbor search.
  LSIMethod = 1
)`

The error:
`2021-05-11 19:30:57 : scATAC_CD34_BMMC_R1 (2 of 3) :  Computing Doublet Statistics, 4.574 mins elapsed.

************************************************************
2021-05-11 19:31:03 : ERROR Found in .computeLSI for  
LogFile = ArchRLogs/ArchR-addDoubletScores-6986e122389-Date-2021-05-11_Time-19-26-22.log

<simpleError in irlba::irlba(mat, nDimensions, nDimensions): max(nu, nv) must be strictly less than min(nrow(A), ncol(A))>

************************************************************`

But the first file scATAC_BMMC_R1, I think it worked well. 
I'm not sure what happened. Here is the log:
[ArchR-addDoubletScores-6986e122389-Date-2021-05-11_Time-19-26-22.log](https://github.com/GreenleafLab/ArchR/files/6459294/ArchR-addDoubletScores-6986e122389-Date-2021-05-11_Time-19-26-22.log)

Thanks a lot. Have a nice day!

**Comment by rcorces** [comment]
Can you post exactly which commands you have used from the start?

**Reply by Tutu-Shi** [reply]
Thank you for your reply. Here it is
```
library(ArchR)
set.seed(1)
addArchRThreads(threads = 6) 
inputFiles <- getTutorialData("Hematopoiesis")#I downloaded these 3 files via my browser because of some connection problems
addArchRGenome("hg19")#install.packages("/mnt/d/R/BSgenome.Hsapiens.UCSC.hg19_1.4.0.tar.gz", repos = NULL)

ArrowFiles <- createArrowFiles(
  inputFiles = inputFiles,
  sampleNames = names(inputFiles),
  minTSS = 4, #Dont set this too high because you can always increase later
  minFrags = 1000, 
  addTileMat = TRUE,
  addGeneScoreMat = TRUE
)
ArrowFiles
[1] "scATAC_BMMC_R1.arrow"      "scATAC_CD34_BMMC_R1.arrow" "scATAC_PBMC_R1.arrow"   #output

doubScores <- addDoubletScores(
  input = ArrowFiles,
  k = 10, #Refers to how many cells near a "pseudo-doublet" to count.
  knnMethod = "UMAP", #Refers to the embedding to use for nearest neighbor search.
  LSIMethod = 1
)

```
Thanks again

**Reply by Tutu-Shi** [reply]
I ran these 3 files one by one. Only scATAC_BMMC_R1 can work.

**Reply by rcorces** [reply]
> I downloaded these 3 files via my browser because of some connection problems

I can only imagine that this is your problem.

**Reply by Tutu-Shi** [reply]
I downloaded them from https://jeffgranja.s3.amazonaws.com/ArchR/TestData/HemeFragments/scATAC_CD34_BMMC_R1.fragments.tsv.gz
https://jeffgranja.s3.amazonaws.com/ArchR/TestData/HemeFragments/scATAC_PBMC_R1.fragments.tsv.gz
scATAC_CD34_BMMC_R1.fragments.tsv.gz 152,214KB 
scATAC_PBMC_R1.fragments.tsv.gz 129,071KB
And the size of these three files sums 461MB. I think it match what tutorial says _The tutorial data is approximately 0.5 GB in size._

I ran my files with ArchR, and it worked well （amazing tools!)

Thanks for your reply.
Have a nice day!
